lib/protocol/schema/provider.capnp

```diff
@@ -11,4 +11,6 @@ interface Publications {
     }
 
     list @3 (owner :Data) -> (publications :List(Text));
+    deals @4 (ns :Text, rel :Text, limit :UInt32, offset :Text) -> (deals :List(import "./definitions.capnp" .DealInfo));
```

Comment:
new RPC methods. offset is sent as text because there's no big int or int256 in capnp

Reply:
yeah, going to stick with uint64